azure/Guidelines.md

```diff
@@ -993,6 +995,14 @@ Client libraries are required to send telemetry and distributed tracing informat
 - [Azure SDK Distributed tracing policy](https://azure.github.io/azure-sdk/general_azurecore.html#distributed-tracing-policy)
 - [Open Telemetry](https://opentelemetry.io/)
 
+In addition to distributed tracing, Azure also uses a set of common correlation headers:
+
+|Name                         |Applies to|Description|
+|-----------------------------|----------|-----------|
+|x-ms-client-request-id.      |Both      |Optional. Caller-specified value identifying the request, in the form of a GUID with no decoration such as curly braces (e.g. `x-ms-client-request-id: 9C4D50EE-2D56-4CD3-8152-34347DC9F2B0`). If the caller provides this header the service **must** log this with their traces to facilitate tracing a single request. Because this header can be client-generated, it should not be assumed to be unique by the service implementation.
+|x-ms-return-client-request-id|Request.  |Optional. If specified, the service **must** include a `x-ms-client-request-id` header with the corresponding value in the response. Allows clients to correlate responses with requests.
```

Comment:
If `true` or `false` value specified, the service ...

Reply:
I don't think this request header is useful. Why not just always return the x-ms-client-request-id and simplify what a service has to implement? If the client doesn't send the ID, then the service doesnt' have to return it (since it doesn[t even have it).

Reply:
This is an existing header recommendation (see aka.ms/rpapiv2). We can chose to not recommend its use and I would be fine with that.

Reply:
I see. My preference would be to not mention this header at all

Reply:
I also prefer to drop this header. There is no harm in returning `x-ms-client-request-id` if the caller passed it, so for the sake of simplicity we should recommend that it be returned whenever it was specified on the request.

Reply:
Removed